internal/postprocessor/config.yaml

```diff
@@ -0,0 +1,776 @@
+   service-config: datastore_v1.yaml
+ - input-directory: google/devtools/artifactregistry/v1
+   service-config: artifactregistry_v1.yaml
+ - input-directory: google/devtools/artifactregistry/v1beta2
+   service-config: artifactregistry_v1beta2.yaml
+ - input-directory: google/devtools/cloudbuild/v1
+   service-config: cloudbuild_v1.yaml
+ - input-directory: google/devtools/cloudbuild/v2
+   service-config: cloudbuild_v2.yaml
+ - input-directory: google/devtools/clouddebugger/v2
+   service-config: clouddebugger_v2.yaml
+ - input-directory: google/devtools/clouderrorreporting/v1beta1
+   service-config: clouderrorreporting_v1beta1.yaml
+ - input-directory: google/devtools/cloudtrace/v1
+   service-config: cloudtrace_v1.yaml
+ - input-directory: google/devtools/cloudtrace/v2
+   service-config: cloudtrace_v2.yaml
+ - input-directory: google/devtools/containeranalysis/v1beta1
+   service-config: containeranalysis_v1beta1.yaml
+ - input-directory: google/devtools/containeranalysis/v1beta1/grafeas
+   service-config: ../containeranalysis_v1beta1.yaml
+ - input-directory: google/firestore/admin/v1
+   service-config: firestore_v1.yaml
+ - input-directory: google/firestore/v1
+   service-config: firestore_v1.yaml
+ - input-directory: google/iam/credentials/v1
+   service-config: iamcredentials_v1.yaml
+ - input-directory: google/iam/v1
+   service-config: iam_meta_api.yaml
+ - input-directory: google/iam/v2
+   service-config: iam_v2.yaml
+ - input-directory: google/identity/accesscontextmanager/v1
+   service-config: accesscontextmanager_v1.yaml
+ - input-directory: google/logging/v2
+   service-config: logging_v2.yaml
+ - input-directory: google/longrunning
+   service-config: longrunning.yaml
+ - input-directory: google/maps/addressvalidation/v1
+   service-config: addressvalidation_v1.yaml
+ - input-directory: google/maps/mapsplatformdatasets/v1alpha
+   service-config: mapsplatformdatasets_v1alpha.yaml
+ - input-directory: google/maps/routing/v2
+   service-config: routes_v2.yaml
+ - input-directory: google/monitoring/dashboard/v1
+   service-config: monitoring.yaml
+ - input-directory: google/monitoring/metricsscope/v1
+   service-config: monitoring.yaml
+ - input-directory: google/monitoring/v3
+   service-config: monitoring.yaml
+ - input-directory: google/privacy/dlp/v2
+   service-config: dlp_v2.yaml
+ - input-directory: google/pubsub/v1
+   service-config: pubsub_v1.yaml
+ - input-directory: google/spanner/admin/database/v1
+   service-config: spanner.yaml
+ - input-directory: google/spanner/admin/instance/v1
+   service-config: spanner_admin_instance.yaml
+ - input-directory: google/spanner/v1
+   service-config: spanner.yaml
+ - input-directory: google/storage/v2
+   service-config: storage_v2.yaml
+ - input-directory: google/storagetransfer/v1
+   service-config: storagetransfer_v1.yaml
+   # All entries below are unmigrated clients. We need to store extra
+   # info for them since they don't have an entry in owlbot.yaml yet.
```

Comment:
Do some of the unmigrated clients keys shadow entries above this line? For example, 'google/devtools/cloudtrace/v1'?

Reply:
Oops you are right, I generated the initial set before I realized non-migrated clients needs more info. Updated